notebooks/nuextract-structure-extraction/nuextract-structure-extraction.ipynb

```diff
@@ -0,0 +1,824 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #1.    core = ov.Core()
</pre><p>could you please use device_widget  <a href="https://github.com/openvinotoolkit/openvino_notebooks/blob/latest/utils/notebook_utils.py#L31" target="_blank">https://github.com/openvinotoolkit/openvino_notebooks/blob/latest/utils/notebook_utils.py#L31</a></p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/openvinotoolkit/openvino_notebooks/pull/2311/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='12'/>

Reply:
<p>Fixed</p>